File: codeforces/136A/cpp/136a.cpp
#include <iostream>
using namespace std;

const int c_max = 128;

int main()
{
    int n, f;
    cin >> n;

    int result[c_max] = {0};
    for (int i = 1; i <= n; i++) {
        cin >> f;
        result[f] = i;
    }

    for (int i = 1; i <= n; i++) {
        cout << result[i] << " ";
    }
    cout << endl;

    return 0;
}

File: codeforces/136A/go/136a.go
package main

import "fmt"

const c_max = 128

func main() {
    var n, f int
    var result[c_max] int
    fmt.Scanf("%d\n", &n)
    for i := 1; i < n; i++ {
        fmt.Scanf("%d", &f)
        result[f] = i
    }
    fmt.Scanf("%d\n", &f)
    result[f] = n

    for i := 1; i <= n; i++ {
        fmt.Printf("%d ", result[i])
    }
    fmt.Print("\n")
}
